curriculum/challenges/english/25-front-end-development/quiz-react-basics/66f1a2009e65c9a40a26d51e.md

```diff
@@ -17,439 +17,438 @@ Answer all of the questions below correctly to pass the quiz.
+Managing component styles.
 
 ---
 
-Placeholder distractor 2
+Passing data through the component.
 
 ---
 
-Placeholder distractor 3
+Controlling component re-renders.
 
 #### --answer--
 
-Placeholder answer
+Passing data through the component tree without props.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+How do you initialize a state variable in a functional component?
 
 #### --distractors--
 
-Placeholder distractor 1
+const count = 0;
 
 ---
 
-Placeholder distractor 2
+const [count, setCount] = useEffect("");
 
 ---
 
-Placeholder distractor 3
+const state = 0;
 
 #### --answer--
 
-Placeholder answer
+const [count, setCount] = useState(0);
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is a con of using Vanilla JS instead of a framework?
 
 #### --distractors--
 
-Placeholder distractor 1
+Automatic state management.
 
 ---
 
-Placeholder distractor 2
+Built-in routing capabilities.
 
 ---
 
-Placeholder distractor 3
+Declarative UI rendering.
 
 #### --answer--
 
-Placeholder answer
+Increased verbosity and manual DOM manipulation.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+How does a single-page application typically update the view?
 
 #### --distractors--
 
-Placeholder distractor 1
+By making a full server reload.
 
 ---
 
-Placeholder distractor 2
+By reloading the entire HTML document.
 
 ---
 
-Placeholder distractor 3
+By using server-side rendering only.
 
 #### --answer--
 
-Placeholder answer
+By updating only the necessary parts of the page.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the correct way to create a functional React component?
 
 #### --distractors--
 
-Placeholder distractor 1
+Using classes only.
 
 ---
 
-Placeholder distractor 2
+Using the React.Component only.
 
 ---
 
-Placeholder distractor 3
+Using JavaScript constructors.
 
 #### --answer--
 
-Placeholder answer
+Using the function keyword or arrow functions.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following is not a valid JSX element?
 
 #### --distractors--
 
-Placeholder distractor 1
+`<div className='class'>`
 
 ---
 
-Placeholder distractor 2
+`<span>Hello World</span>`
 
 ---
 
-Placeholder distractor 3
+`<button>Click Me</button>`
 
 #### --answer--
 
-Placeholder answer
+`<img class='img'>`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following is an example of a meta framework for React?
 
 #### --distractors--
 
-Placeholder distractor 1
+Vue.js
 
 ---
 
-Placeholder distractor 2
+Angular
 
 ---
 
-Placeholder distractor 3
+Svelte
 
 #### --answer--
 
-Placeholder answer
+Next.js
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What tool would you use to bootstrap a new React project?
 
 #### --distractors--
 
-Placeholder distractor 1
+Webpack
 
 ---
 
-Placeholder distractor 2
+npm
 
 ---
 
-Placeholder distractor 3
+Babel
 
 #### --answer--
 
-Placeholder answer
+create-react-app
```

Comment:
`create-react-app` is so outdated now that even the React docs no longer recommend it.